Add instruction to restart Storybook server after addon configuration… (#764)
chromaui/learnstorybook.com: content/intro-to-storybook/angular/en/simple-component.md

```

> After updating your Storybook configuration file (`.storybook/main.ts`), restart the Storybook server to apply the new addon configuration and have the UI updated accordingly. Run the following command again:

Reviewer: @MasakiNishi, we have a bit of room for improvement here and can simplify this to the following:

```suggestion
Finally, restart your Storybook to see the new addon enabled in the UI.
```

Also, remove the `yarn storybook` shell command

Author: @jonniebigodes, Thank you for the suggested improvements. I have applied them to other versions as well.
Since the changes were also incorporated into the Vue files, I have applied them to only two files.
